fapi-client/src/main/scala/com/gu/facia/api/utils/ItemKicker.scala

```diff
@@ -89,6 +89,17 @@ object ItemKicker {
     case FreeHtmlKickerWithLink(body, _) => Some(body)
     case _ => None
   }
+
+  /** Return the canonical display text for a kicker */
+  def kickerDisplayText(itemKicker: ItemKicker): Option[String] = itemKicker match {
```

Comment:
Yeah, I thought that might be the reason BUT. If we don't support HTML that's our problem so maybe it doesn't need to be in the Facia client (we could just strip it, unless the HTML is complex and meaningful - you'll know best on this score)?

If that's the case though (and you think this is helpful in the client) then the function above should probably be renamed to `kickerHTML` and this one should be `kickerText`.